src/models/data-cube/filters.ts

```diff
@@ -26,3 +26,15 @@ export function inOneOfDateRanges(dateRanges: [Date, Date][], rangeOptions: Rang
       });
   };
 }
+
+export function inCities(cityIds: string[]): Filter {
+  return (categories: Category[]) => {
+    const cityIndex = categories.findIndex(
+      category => category.name === 'city',
+    );
+    return (row: Row) => {
+      const cityId = row.header[cityIndex] as string;
```

Comment:
why typecast? Should you coerce any numbers into strings instead?

Reply:
`DataCube` is designed to have `string | number` as a header, but city header is always `string`.